### ISSUE TITLE
Potential Issue: Rain frequency is FAR too high

### ISSUE THREAD

**Issue by Teacyn** [Teylor-SG/RGE-Bug-Reports#7]
**Describe the bug**
Something I've noticed is that it seems to rain pretty much 80% of the time. I've been in-game for about 4 weeks now (~15 real life hours) and it has not stopped raining for more than 8-9 ingame hours at any given time, in any region of the world.

**Expected behavior**
For Skyrim not to be a Rainforest.

**Comment by jdsmith2816** [comment]
I haven't noticed an abundance of rain.  Are you still seeing this?

**Comment by Teacyn** [comment]
Yes, I am still seeing this. I am a few updates behind now (still post 2.0 though), but I can confirm I'm still getting rain almost every single day in-game, in pretty much any area of the game.
